feel/feeldiscr/functionspace.hpp

```diff
@@ -4313,38 +4313,55 @@ public std::enable_shared_from_this<FunctionSpace<A0,A1,A2,A3,A4> >
                    size_type mesh_components = MESH_RENUMBER | MESH_CHECK,
                    periodicity_type  periodicity = periodicity_type(),
                    worldscomm_ptr_t const& _worldsComm = Environment::worldsComm(nSpaces),
-                   std::vector<bool> extendedDofTable = std::vector<bool>(nSpaces,false) )
+                   std::vector<bool> extendedDofTable = std::vector<bool>(nSpaces,false),
+                   const std::string& name = functionSpaceDefaultInstanceName() )
         :
         super( _worldsComm[0]->clone() ),
         M_worldsComm( _worldsComm ),
         M_extendedDofTableComposite( extendedDofTable ),
         M_extendedDofTable( extendedDofTable[0] )
     {
+        FUNCTIONSPACE_INSTANCE_NUMBER++;
```

Comment:
coding style.
and I think it's no very nice to use a global variable.
We can use a uuid if the name is not given

Reply:
Currently, it's not an identifier, but more the index of the object instance.
Rather than uuid, I think it's better to guaranty that 2 simulations produce the same "name-<index>" by default if the user has not precised the instance name.

I agree for the global var, I will change that.